### DISCUSSION TITLE
New instance of an app opens alongside same app

### DISCUSSION THREAD

**Discussion by barbas66** [JaKooLit/Hyprland-Dots#165] [Q&A]
Hi there,

I'm a newcomer, so, please, be patient!

Here's what's happening in my system: I have an app (say Firefox) open in a workspace (say workspace 1). When I run another Firefox instance from any workspace (say 3), its window ends up in the workspace 1 where the first instance is running, not in the workspace 3 I ran it from (see picture).

Is this an expected behaviour? How can i get rid of this?

Thanks for your help!

PS. This doesn't happen with the terminal.

![Screenshot_20-feb_21-01-50_26872](https://github.com/JaKooLit/Hyprland-Dots/assets/108096320/4cc04808-a312-4a96-b325-a750948c0ba1)

**Comment by JaKooLit** [comment]
edit ~/.config/hypr/UserConfigs/WindowRules.conf

https://wiki.hyprland.org/Configuring/Window-Rules/#window-rules-v2

**Reply by barbas66** [reply]
Ok, thanks!

Solved!
